src/components/Asset/AssetActions/Download.tsx

```diff
@@ -236,7 +236,7 @@ export default function Download({
       disabled={baseTokenBalance === 0 || !baseTokenBalance}
       hasPreviousOrder={false}
       hasDatatoken={false}
-      dtSymbol={asset?.accessDetails?.baseToken.symbol}
```

Comment:
I'm pretty sure we need to use `.baseToken` here because we need the symbol of the base token to be collected by the publisher, not the datatoken, so the needed change would be: `dtSymbol={asset?.accessDetails?.baseToken?.symbol}`, as Matthias suggested